### PR TITLE
feat(rust): pretty json output by default, and colored if possible

### PR DESCRIPTION
**Pull request by adrianbenavides**
- replace `--pretty` by `--compact-output` to mimic the `jq` api
- apply colors to json output if possible (TTY and NO_COLOR is not passed)
- ux improvement: if either `--jq` or `--compact-output` args are passed, the `--output-format json` is assumed

[asciicast](https://asciinema.org/a/mB8T2sepaLHcqtT7ZevghwYME)

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/global_args.rs [hunk 1]
```diff
@@ -36,85 +34,55 @@ pub struct GlobalArgs {
...
     /// jq query to apply to the JSON output of the command
     #[arg(global = true, long = "jq")]
-    pub jq_query: Option<String>,
+    jq_query: Option<String>,
 
-    /// Pretty print the JSON output of the command
+    /// Compact the JSON output of the command
     #[arg(global = true, long)]
-    pub pretty: bool,
+    compact_output: bool,
```
Comment (davide-baldo) [inline implementations/rust/ockam/ockam_command/src/global_args.rs:61]: Should we add an alias for `--pretty` to keep back-compatibility?

**Review by davide-baldo — DISMISSED** [review]
Looks good to me!

File: implementations/rust/ockam/ockam_command/src/global_args.rs (hunk 1 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/global_args.rs:61]: Yeah good idea, but it will have to be a separate argument. I'll add it in a bit

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/global_args.rs:61]: ready now!

**Review by etorreborre — APPROVED** [review]

**Pull request merged**